public/index.html

```diff
@@ -1,11 +1,45 @@
 <!DOCTYPE html>
 <html>
+
 <head>
   <meta charset="utf-8">
-  <title>13-netflix-result</title>
+  <title>13-ntflix-result</title>
   <link rel="stylesheet" href="/app.css">
 </head>
+
 <body>
+  <div class="section">
+    <div class="container">
+      <div class="content">
+        <div class="top">
+          <input type="search" class="search" value="Movie Title">
```

Comment:
Use `placeholder` not value to prompt the user otherwise you will be filling in this input and the user will have to delete what's already there instead of typing something in themselves